tests/test_google_ads_automatic_fields.py

```diff
@@ -78,7 +78,9 @@ def test_happy_path(self):
         conn_id = connections.ensure_connection(self)
 
         streams_to_test = {stream for stream in self.expected_streams()
-                           if not self.is_report(stream)}
+                           if not self.is_report(stream)} - {
+                                   "call_details"
```

Comment:
could you just a dd a comment explaining this is skipped because we have insufficient test data (rather than it not being applicable).